services/src/test/scala/cromwell/pipeline/service/AggregatorServiceTest.scala

```diff
@@ -0,0 +1,74 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dao.repository.utils.{ TestProjectUtils, TestRunUtils, TestUserUtils }
+import cromwell.pipeline.datastorage.dto.{
+  CromwellInput,
+  FileParameter,
+  FileTree,
+  PipelineVersion,
+  Project,
+  ProjectConfiguration,
+  ProjectFile,
+  ProjectFileConfiguration,
+  ProjectFileContent,
+  StringTyped
+}
+import org.mockito.Mockito.when
+import org.scalatest.{ AsyncWordSpec, Matchers }
+import org.scalatestplus.mockito.MockitoSugar
+
+import java.nio.file.Paths
+import scala.concurrent.Future
+
+class AggregatorServiceTest extends AsyncWordSpec with Matchers with MockitoSugar {
+
+  private val projectService: ProjectService = mock[ProjectService]
+  private val projectConfigurationService: ProjectConfigurationService = mock[ProjectConfigurationService]
+  private val projectVersioning: GitLabProjectVersioning = mock[GitLabProjectVersioning]
+  private val aggregatorService: AggregatorService =
+    new AggregatorService(projectService, projectVersioning, projectConfigurationService)
+
+  "AggregatorService" when {
```

Comment:
I think we need more tests. With all kind of different results, when things go wrong.